test/tests/outputs/console/tests

```diff
@@ -161,6 +161,6 @@
     type = RunException
     input = console.i
     cli_args = Outputs/screen/fit_mode=foo
-    expect_err = "Failed to convert 'foo' to an int."
+    expect_err = "Unable to convert foo to type int"
```

Comment:
Minor: I really like when these numbers/strings are quoted. This PR looks fine, but I'd suggest restoring these.